spec/features/user_view_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+RSpec.describe "User view" do
+  context 'with valid attributes' do
+    
+    it 'can create a new user' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.